python/metatensor-core/tests/serialization.py

```diff
@@ -80,6 +80,30 @@ def test_load(use_numpy, memory_buffer, standalone_fn):
     assert gradient.values.shape == (59, 3, 5, 3)
 
 
+@pytest.mark.parametrize("use_numpy", (True, False))
+def test_load_deflate(use_numpy):
+    # This file was saved using DEFLATE to compress the different ZIP archive members
```

Comment:
I am just wondering how we were able to load this file before without supporting _deflate_?

Reply:
we could only load it with `use_numpy=True`, which does support compression. Instead of this test, I could also do `use_numpy=False` when loading data in operations tests. What do you think?

Reply:
Yes makes sense. Now I see the parametrize.